topic-operator/src/main/java/io/strimzi/operator/topic/Session.java

```diff
@@ -37,6 +38,9 @@
 
     private final static Logger LOGGER = LogManager.getLogger(Session.class);
 
+    private final static String SASL_TYPE_PLAIN = "PLAIN";
+    private final static String SASL_TYPE_SCRAM_SHA_512 = "PLAIN";
```

Comment:
Is this a typo? Should it be `SCRAM-SHA-512`?

Reply:
Yes, typo.